### DISCUSSION TITLE
下载显示内容仍处于加密状态

### DISCUSSION THREAD

**Discussion by Namjw11** [pjy612/SteamManifestCache#15] [Question / Answer]
按照wiki里的步骤后没办法下载

**Comment by pjy612** [comment]
很大概率是缺密钥。
特别是解锁了带容量的DLC的时候。如果 DLC 对应的 密钥没有就会这样。
其次就是 没入正 却订阅创意工坊。

**Reply by Namjw11** [reply]
不知道为什么又能下载了，缺密钥的话直接显示是空包。

**Reply by pjy612** [reply]
SAI 你缺密钥的话 直接就不帮你解锁你缺的部分。
但是如果你自己拖了 DLC啥的，那个DLC还带内容 可能就会出现 卡下载或卡更新（内容加密）
